docs/review/review.yaml

```diff
@@ -20,19 +20,75 @@ paths:
                 - _id: 6255bc080a75adf9223df100
                   comment: Not bad
                   rating: 4
-                  authorId: 6255bc080a75adf9223df211
+                  author:
+                    _id: 6255bc080a75adf9223df100
+                    role: [ student, tutor ]
+                    firstName: John
+                    lastName: Doe
+                    email: johndoe@gmail.com
+                    categories:
+                      [
+                        { _id: 6255bc080a75adf9223df444, name: Languages },
+                        { _id: 6255bc080a75adf443d7644, name: Biology }
+                      ]
+                    totalReviews: 10
+                    averageRating: 4.5
+                    nativeLanguage: english
+                    address: { country: The USA, city: California }
+                    education: KNPU H.S. Skovoroda
+                    photo: john-doe-photo.jpg
+                    lastLogin: null
+                    createdAt: 2021-04-09T11:34:53.243+00:00
+                    updatedAt: 2022-09-02T11:59:53.243+00:00
                   targetUserId: 6255bc080a75adf9223df212
                   targetUserRole: tutor
-                  offerId: 6255bc033a75adf9223df231
+                  offer:
+                    _id: 63ebc6fbd2f34037d0aba791
+                    price: 111
+                    proficiencyLevel: Beginner
+                    description: this is a new blbsksfmlgksjfglk jlskgjslfgkjsflgk l gks lgsl sjglkgj
```

Comment:
it's nothing, but be so kind if you can and not quite lazy, make it look pretty, add 'course' or something after 'a new' instead of this gibberish

Reply:
fixed

Reply:
thankie, you saved my eyes ^^